README.md

```diff
@@ -1,214 +1,201 @@
-- Ensure the OPENFN_REPO_DIR env var is set to to the root of this repo, ie,
-  `export OPENFN_REPO_DIR=~/repo/openfn/cli-repo`
-- Ensure you have the logo images of the adaptor you need to create: a rectangle
-  512x190px and a square-256x256px
+#### Prerequisites
 
-#### 1. Setup
+1. Set up the Repository Directory
 
-Generating a new adaptor. Run this from the repo root:
+   The CLI needs a designated folder to save and load adaptors. Configure this
+   by setting the `OPENFN_REPO_DIR` environment variable.
 
-```
-pnpm generate <adaptor-name>
-```
+   Add to your shell configuration file (e.g., `~/.bashrc`, `~/.zshrc`):
 
-- Add the logo images in the assets folder inside the generated adaptor folder.
-  Ps: Ensure both images are named rectangle.png and square.png respectively and
-  adhere to the size specifications mentioned in the requirements section.
-- Ensure the mages have a transparent background. Navigate to
-  configuration-schema.json, and change any configs that do not align with the
-  adaptor
-- Go to `/src/Adaptor.js` and create the adaptor’s Operations - the functions
-  used in job code. You may want to set up `POST, GET,` to fit the current
-  adaptor’s requirements
-
-They should look something like this:
-
-```js
-export function yourFunctionName(arguments) {
-  return state => {
-    // logic
-    return state;
-  };
-}
-```
+   ```bash
+   # Directory for CLI adaptor storage
+   export OPENFN_REPO_DIR=~/repo/openfn/cli-repo
 
-- Go to `src/Utils.js` and change the code in the file to match your desired
-  implementation. Any internal functions used by your adaptor, but not by job
-  code, should go here in Utils.js
-- Update the readme with correct examples that match your new implementation
-- Write and update the tests within the `/tests` folder
-- Edit the CHANGELOG.md file with comments about the initial release.
-- You should set your credentials in the “configuration” property inside
-  `state.json`
-- Write some unit tests to ensure your code works
+   # Optional: For development with adaptors monorepo
+   export OPENFN_ADAPTORS_REPO=~/repo/openfn/adaptors
 
-#### 2. Test it manually
+   ```
 
-To run a test job and exercise your
+#### Development Steps
 
-- Create a new /tmp folder and create two new files: state.json and
-  expression.js.
-- - Note that the /tmp folder is already gitignored and will not be sent to
-    gitHub when you push your code.
-- Write a test job inside `expression.js`
-- Navigate to the adaptor folder:
+1. Generate the adaptor structure:
 
-```
-cd packages/<adaptor name>
-```
+   ```bash
+   pnpm generate <adaptor-name>
+   ```
 
-- Install adaptor dependencies
+2. Add your logo images to `packages/<adaptor-name>/assets/`:
 
-```
-pnpm install
-```
+   - `rectangle.png` (512x190px)
+   - `square.png` (256x256px)
 
-- Build the adaptor
+3. Implement your adaptor in `packages/<adaptor-name>/src/Adaptor.js`
 
-```
-pnpm build --watch
-```
+4. Test your adaptor:
```

Comment:
I would rename this to "build your adaptor". And maybe add an optional section to add tests.

I should update the article but probably helpful to link to the unit testing guide here: https://github.com/OpenFn/adaptors/wiki/Unit-Testing-Guide